NangoHQ/nango
# fix(CLI): show on-events scripts in `nango deploy` confirmation message

Depends on https://github.com/NangoHQ/nango/pull/3067

The CLI should show the added/deleted on-event scripts in the `nango deploy` confirmation message.

<img width="1106" alt="Screenshot 2024-11-26 at 15 52 20" src="https://github.com/user-attachments/assets/787b394d-05b2-4833-938e-07b3683c1501">

How to test:
1. Install local CLI: `nango install -g PATH/TO/REPO/packages/cli`
2. Add/Remove `on-events` scripts in your nango.yaml
3. run `nango deploy dev` and check the confirmation message show the added/removed scripts
